frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -1,5 +1,11 @@
 import cx from "classnames";
-import { useEffect, useRef, useCallback, type ReactNode } from "react";
+import {
+  useEffect,
+  useRef,
```

Comment:
Probably a question for the product team:

Should we show the "View all functions" button when user is trying to reference a field (i.e. when user types `[` in the custom expression)?

![image](https://github.com/metabase/metabase/assets/6830683/600512f1-aa21-445b-b1ac-d68b5b98f9d0)

Reply:
Yeah, let's remove this View all... link when there is at least one symbol in the input
It was intended to show this link only in initial state
If it is shown when the customer already entered something, we have double scrollbar, and overall it makes less sense to show it
<img width="470" alt="Captura de ecrã 2024-04-15, às 14 20 00" src="https://github.com/metabase/metabase/assets/777800/5cfb9464-3234-4ac8-a90f-c9cb4bf324b0">